ui/src/index.js

```diff
@@ -1,10 +1,24 @@
+import { createStore } from "redux";
+import { Provider } from "react-redux";
 import React from "react";
 import ReactDOM from "react-dom";
-import "./index.css";
-import App from "./App";
+
+import "./scss/base.scss";
 import * as serviceWorker from "./serviceWorker";
+import App from "./app/App";
+import reducers from "./root-reducer";
+
+const store = createStore(
+  reducers,
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
Maybe leave a comment here that once we're using redux middleware like sagas, we'll probably want to change this to use the [redux-devtools-extension](https://github.com/zalmoxisus/redux-devtools-extension#13-use-redux-devtools-extension-package-from-npm).

Reply:
Oh nice!